pkg/operation/botanist/controlplane/kubescheduler/monitoring.go

```diff
@@ -0,0 +1,155 @@
+	monitoringMetricRestClientRequestsTotal                               = "rest_client_requests_total"
+	monitoringMetricProcessMaxFds                                         = "process_max_fds"
+	monitoringMetricProcessOpenFds                                        = "process_open_fds"
+
+	monitoringAlertingRules = `groups:
+- name: kube-scheduler.rules
+  rules:
+  - alert: KubeSchedulerDown
+    expr: absent(up{job="` + monitoringPrometheusJobName + `"} == 1)
+    for: 15m
+    labels:
+      service: ` + ServiceName + `
+      severity: critical
+      type: seed
+      visibility: all
+    annotations:
+      description: New pods are not being assigned to nodes.
+      summary: Kube Scheduler is down.
+
+  ### Scheduling latency ###
+  - record: cluster:scheduler_e2e_scheduling_latency_seconds:quantile
+    expr: histogram_quantile(0.99, sum(` + monitoringMetricSchedulerE2eSchedulingLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.99"
+  - record: cluster:scheduler_e2e_scheduling_latency_seconds:quantile
+    expr: histogram_quantile(0.9, sum(` + monitoringMetricSchedulerE2eSchedulingLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.9"
+  - record: cluster:scheduler_e2e_scheduling_latency_seconds:quantile
+    expr: histogram_quantile(0.5, sum(` + monitoringMetricSchedulerE2eSchedulingLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.5"
+  - record: cluster:scheduler_scheduling_algorithm_latency_seconds:quantile
+    expr: histogram_quantile(0.99, sum(` + monitoringMetricSchedulerSchedulingAlgorithmLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.99"
+  - record: cluster:scheduler_scheduling_algorithm_latency_seconds:quantile
+    expr: histogram_quantile(0.9, sum(` + monitoringMetricSchedulerSchedulingAlgorithmLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.9"
+  - record: cluster:scheduler_scheduling_algorithm_latency_seconds:quantile
+    expr: histogram_quantile(0.5, sum(` + monitoringMetricSchedulerSchedulingAlgorithmLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.5"
+  - record: cluster:scheduler_binding_latency_seconds:quantile
+    expr: histogram_quantile(0.99, sum(` + monitoringMetricSchedulerBindingLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.99"
+  - record: cluster:scheduler_binding_latency_seconds:quantile
+    expr: histogram_quantile(0.9, sum(` + monitoringMetricSchedulerBindingLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.9"
+  - record: cluster:scheduler_binding_latency_seconds:quantile
+    expr: histogram_quantile(0.5, sum(` + monitoringMetricSchedulerBindingLatencyMicrosecondsBucket + `) BY (le, cluster)) / 1e+06
+    labels:
+      quantile: "0.5"`
```

Comment:
Can we have a newline at the end of every long string?
```suggestion
      quantile: "0.5"
`
```